stories/src/band.stories.tsx

```diff
@@ -274,7 +274,7 @@ storiesOf('Band Chart', module)
     const shadeOpacity = number('Shade Opacity', 0.3)
     const hoverDimension = select(
       'Hover Dimension',
-      {auto: 'auto', x: 'x', y: 'y', xy: 'xy', none: ''},
+      {auto: 'auto', x: 'x', y: 'y', xy: 'xy'},
```

Comment:
is this necessary or a leftover change from previous testing?

Reply:
went ahead and removed this